src/firefly/js/templates/lightcurve/LcViewer.jsx

```diff
@@ -96,7 +96,7 @@ export class LcViewer extends PureComponent {
         const LcPeriodInstance=  get(getAppOptions(), 'charts.chartEngine')==='plotly' ? LcPeriodPlotly : LcPeriod;
 
         var mainView = (err,converterId) => {
-            if (!isEmpty(error) && converterId) {
+            if (error!='undefined' && !isEmpty(error) && converterId) {
 
```

Comment:
i thought `isEmpty` functionn from lodash covered the case ```error!='undefined'```, is not? WOW, I'm surprised! i've used elsewhere wrongly then. @loitly @robyww did you know that?